subprojects/docs/src/main/resources/header.html

```diff
@@ -95,92 +95,78 @@
-                    <li><a href="/userguide/intro_multi_project_builds.html">Multi-Project Builds</a></li>
+            <li>Using the Command Line (TBD)</li>
+            <li>Using Gradle From an IDE (TBD)</li>
+            <li><a class="nav-dropdown" data-toggle="collapse" href="#customizing-execution" aria-expanded="false" aria-controls="customizing-execution">Customizing Execution</a>
+                <ul id="customizing-execution">
+                    <li><a href="/userguide/build_environment.html">Configuring the Build Environment</a></li>
+                    <li><a href="/userguide/gradle_daemon.html">Configuring the Gradle Daemon</a></li>
+                    <li><a href="/userguide/init_scripts.html">Creating Initialization Scripts</a></li>
                 </ul>
             </li>
+            <li><a href="/userguide/intro_multi_project_builds.html">Executing Multi-Project Builds</a></li>
+            <li><a href="https://guides.gradle.org/creating-build-scans/">Inspecting Gradle Builds</a></li>
             <li><a class="nav-dropdown" data-toggle="collapse" href="#optimizing-build-performance" aria-expanded="false" aria-controls="optimizing-build-performance">Optimizing Build Times</a>
                 <ul id="optimizing-build-performance">
                     <li><a href="https://guides.gradle.org/performance/">Build Performance Guide</a></li>
                     <li><a href="/userguide/build_cache.html">Enabling and Configuring the Build Cache</a></li>
-                    <li><a href="https://guides.gradle.org/using-build-cache/">Using the Build Cache</a></li>
                 </ul>
             </li>
-            <li><a class="nav-dropdown" data-toggle="collapse" href="#customizing-execution" aria-expanded="false" aria-controls="customizing-execution">Customizing Execution</a>
-                <ul id="customizing-execution">
-                    <li><a href="/userguide/build_environment.html">Configuring the Build Environment</a></li>
-                    <li><a href="/userguide/gradle_daemon.html">Configuring the Gradle Daemon</a></li>
-                    <li><a href="/userguide/init_scripts.html">Initialization Scripts</a></li>
-                </ul>
-            </li>
-
-            <li><a href="/userguide/gradle_wrapper.html">Provisioning Gradle (Wrapper)</a></li>
-            <li><a href="/userguide/troubleshooting.html">Troubleshooting</a></li>
+            <li><a href="/userguide/composite_builds.html">Integrating Separate Gradle Builds (Composite Builds)</a></li>
+            <li><a href="/userguide/troubleshooting.html">Troubleshooting builds</a></li>
         </ul>
 
         <h3 id="authoring-gradle-builds">Authoring Gradle Builds</h3>
         <ul>
-            <li><a href="/userguide/groovy_build_script_primer.html">Groovy DSL Primer</a></li>
-            <li><a href="/userguide/kotlin_dsl.html">Kotlin DSL Primer</a></li>
```

Comment:
These don't form part of a narrative that fits well with the user manual. On top of that, they are very different from one another, despite both having "Primer" in their titles.

I've moved them to the appendix as they are useful reference material, but are best introduced via other chapters and content. They certainly shouldn't come before the basics of writing Gradle build scripts I think. In fact, the "basics" chapters should link to them.